Move code from IDR-metadata PR

dominikl opened the pull request:
Moves all the code from https://github.com/IDR/idr-metadata/pull/297 into this separate 'idr0040-aymoz-singlecell' repository.

I also added all my Java 'scripts' I used to prepare the `annotation.csv` and `filePaths.tsv`, and various other scripts (e. g. dealing with the rendering settings) to the `scripts` directory and a brief `scripts/workflow.txt` to explain what I actually did in order to prepare the files.

dominikl committed:
Move code into separate IDR/idr0040-aymoz-singlecell repository

dominikl committed:
Fix yamllint issues

dominikl committed:
Correct small issue in copy_rend_settings.sh

sbesson commented on scripts/rendering/copy_rend_settings.sh:
```diff
@@ -0,0 +1,11 @@
+for i in /uod/idr/filesets/idr0040-aymoz-singlecell/20180215/**/display_and_comments.txt
+do
+	IFS='/' read -ra arr <<< "$i"
+	cp $i /home/dlindner/${arr[6]}_mm_rend.json
```
This probably needs to be update to use relative paths ?

sbesson commented on scripts/annotations/CSVMerger.java:
```diff
@@ -0,0 +1,94 @@
+    // BEGIN Parameters
+    
+    static final String[] files = {
+            "/Users/dlindner/idr0040/idr0000-experimentB-assays_DoseResponsesSupFigs.csv",
```
Should these files be moved under revision control and/or the paths adjusted to point to `/uod/idr/filesets`?

sbesson reviewed (approved):
Overall the annotation aspect has been extensively discussed in https://github.com/IDR/idr-metadata/pull/297 and I would be inclined to get this merged asap as it captures the state of the metadata files used for the import into the testing server.

A coupled of comments on the scripts used for generating the annotation files (:+1: for committing them). Overall there might be a review of all these scripts across repos so that we can start refactor common functionalities upstream.

dominikl committed:
Sanitized paths

dominikl commented:
That's true, changed the paths to relative paths throughout, looks better. And added reference as comment to original location under `/uod/idr/filesets` where appropriate.

dominikl committed:
Explained annotation.csv prep a bit better
